velox/exec/Task.h

```diff
@@ -357,6 +379,11 @@ class Task : public std::enable_shared_from_this<Task> {
       uint32_t splitGroupId,
       const core::PlanNodeId& planNodeId);
 
+  // Returns a Customized Join Bridge for 'planNodeId'.
```

Comment:
typos: "Returns a custom join bridge for 'planNodeId'."

Reply:
Fixed